README.md

```diff
@@ -17,9 +17,10 @@ Hermes is an E2E testing tool created mainly for the use in [ABSA OSS][gh-absa]
 sbt assembly
 ```
 
-Known to work with:
+## Known to work with:
 
-- Spark 2.4.4
+- Spark 2.4.2 - 3.1.1
+  - There are now spark version guards to protect from false positives. If there is someone willing to test for older versions, we are happy to extend these.
 - Java 1.8.0_191-b12
 - Scala 2.11.12
```

Comment:
How are testing Spark 3.1.1 @ Scala 2.11? And if really so, why would you?

Reply:
I am not. It is imposible AFAIk. I have pushed an update to that message